### ISSUE TITLE
Is there fork layer now (multi-task learning)? 

### ISSUE THREAD

**Issue by bryanyzhu** [fchollet/keras#2650]
Hi all,

I am trying to do multi-task learning, like a single network, multiple loss functions. I show a simple graph below, hope you will get it.
```
                                  | --->   layer3_1 ---> loss
layer1 ---> layer2 --->   
                                  | --->   layer3_2 ---> loss
```
I already read through issue 224 [here](https://github.com/fchollet/keras/issues/224), but i can't find the fork function or layer, or split function. Could anybody show me how to do this? Thank you

**Comment by joelthchao** [comment]
This [guide](http://keras.io/getting-started/functional-api-guide/#multi-input-and-multi-output-models) shows how to share layer via functional API and train a network with multiple losses.

**Comment by bryanyzhu** [comment]
Thanks @joelthchao I will look into it.
